straxen/common.py

```diff
@@ -488,11 +490,11 @@ def remap_old(data, targets, works_on_target=''):
         pass
     elif len(data):
         # select the old data and do the remapping for this
-        warn("Correcting data of runs with mis-cabled PMTs. \nSee: https://"
-             "xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenonnt:dsg:daq:sector_swap. "
-             "Don't use '' selection_str='channel == xx' '' (github.com/XENONnT/straxen/issues/239)")
         mask = data['time'] < TSTART_FIRST_CORRECTLY_CABLED_RUN
-        data[mask] = remap_channels(data[mask])
+        res = data.copy()
+        res[mask] = remap_channels(data[mask])
+        data = res.copy()
+        del res
```

Comment:
Could you quickly explain why you have to copy the data twice? I know you would not have added it if it would not be needed, but still it looks odd to me. Other than that it looks fine.

Reply:
Actually it could do with once, good catch, it's needed at least once since it is read-only otherwise.